### PR TITLE
Correct github workflow for gem push

### PR DESCRIPTION
**Pull request by bobmaerten**

### CODE REVIEW

**Comment by czj** [comment]
C'est vrai joli ces diagrammes d'actions !
En plus visuel de https://www.buddy.works !

File: fuel_surcharge.gemspec [hunk 1]
```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features|.github)/}) }
```
Comment (czj) [inline fuel_surcharge.gemspec]: Maybe we should have a `select()` instead of a `reject()` here to be safe and future proof ?

Comment (bobmaerten) [reply fuel_surcharge.gemspec]: Including all except Build/workflow files 
Vs
Include a big list of files and folders

Don’t know... really

**Comment by czj** [comment]
Oh it would be that big ?
I thought like 2 files + 2 folders :-(

**Pull request merged**